app/Nova/Metrics/ActiveAttendanceBreakdown.php

```diff
@@ -27,20 +33,42 @@ public function name(): string
      */
     protected $showAllTime = false;
 
+    /**
+     * If displaying on the main dashboard, this indicates the event to get attendance from.
+     */
+    private int $resourceId = -1;
+
+    /**
+     * If displaying on a page different from the type of attendable you are showing.
+     * This indicates the morphClass of the attendable (for example, an event).
+     */
+    private string $attendableType;
```

Comment:
Make both of these nullable and default to null, so that you can use null coalescing further down - https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op